beestation.dme

```diff
@@ -2642,6 +2642,7 @@
 #include "code\modules\reagents\reagent_containers\hypospray.dm"
 #include "code\modules\reagents\reagent_containers\medspray.dm"
 #include "code\modules\reagents\reagent_containers\patch.dm"
+#include "code\modules\reagents\reagent_containers\mk2.dm"
```

Comment:
Since the mk2 isn't a reagent container itself, it shouldn't really be here.

Reply:
I have it there because it was where the original hypospray was, figured it would be faster to look up the DM file if they were both in the same palce

Reply:
mk2 isn't a very descriptive file name

Reply:
true, i'll swap it to Hypo_MK2

Reply:
Please follow the naming convention of all the other files